src/utils/polymer.dom.html

```diff
@@ -36,6 +36,11 @@
   class EffectiveNodesObserver {
 
     constructor(target, callback) {
+      /** @type {MutationObserver} */
```

Comment:
Question: Why doesn't the type need to be `{MutationObserver=}` or something to say it can be undefined (given that the property is clearly undefined for some time since it doesn't gain a value in the constructor)?  FWIW, these can just be nulled in the ctor if that is more idiomatic.